packages/access-token-jwt/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "@tsconfig/node12/tsconfig.json",
   "compilerOptions": {
+    "useUnknownInCatchVariables": false,
```

Comment:
https://devblogs.microsoft.com/typescript/announcing-typescript-4-4/#using-unknown-in-catch-variables